blocks/library/gallery/index.js

```diff
@@ -119,10 +121,16 @@ registerBlockType( 'core/gallery', {
 		}
 
 		return (
-			<div className={ `blocks-gallery align${ align }` }>
+			<div className={ `blocks-gallery align${ align } columns-${ columns }` }>
 				{ images.map( ( img, i ) => (
 					<GalleryImage key={ i } img={ img } />
 				) ) }
+				{ focus &&
+					<InspectorControls>
+						<label>Columns:</label>
+						<input type="range" min="1" max={ images.length } value={ columns } onChange={ setColumnsNumber } />
```

Comment:
Seems like we should synchronize the max here with the max in CSS. That is, if we have an upper limit on CSS why not set the upper limit here…

```js
max={ Math.min( images, length, MAX_CSS_COLUMNS ) }
```